plugin/pkg/scheduler/factory/factory_test.go

```diff
@@ -302,3 +307,49 @@ func TestBackoff(t *testing.T) {
 		t.Errorf("expected: 1, got %s", duration.String())
 	}
 }
+
+// TestPodFitScheduler tests if a pod with certain annotation should be picked
+// by the scheduler with name "kube-scheduler"
+func TestPodFitScheduler(t *testing.T) {
```

Comment:
Assuming you take my advice on renaming the function itself, then rename this to TestRepsonsibleForPod()